validation/src/test/scala/hmda/validation/rules/ts/validity/V150Spec.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.ts.validity
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.ts.TsEditCheckSpec
+
+class V150Spec extends TsEditCheckSpec {
+  property("valid when contact name and respondent name are different") {
+    forAll(tsGen) { ts =>
+      whenever(ts.contact.name != ts.respondent.name) {
+        ts.mustPass
+      }
+    }
+  }
+
+  property("invalid when contact name and respondent name are the same") {
+    forAll(tsGen) { ts =>
+      val name = ts.respondent.name
+      val invalidContact = ts.contact.copy(name = name)
```

Comment:
I find name = name to be a bit confusing. Maybe something like `name = respondentName`? This is a tiny quibble though.